lib/cookie.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+class Cookie
+  attr_reader :setcookie_string
+  def initialize(str)
+    @setcookie_string = str
+  end
+
+  def to_s
```

Comment:
This method seems to be the core to the whole class, but simply calling it `to_s` doesn't really reveal what it's actually doing. I would suggest giving this a name that reflects what it actually *is*, and then, if you really need to (though I'm not sure that you do), aliasing `to_s` to it.